server/data/orchestrationApiClient.ts

```diff
@@ -210,7 +210,7 @@ export default class OrchestrationApiClient {
     bookerReference: string
   }): Promise<AvailableVisitSessionDto[]> {
     return this.restClient.get({
-      path: '/visit-sessions/available',
+      path: config.features.visitRequest ? '/visit-sessions/available/v2' : '/visit-sessions/available',
```

Comment:
This is to only use the new v2 sessions API if the feature flag is enabled (i.e. in dev & staging)